src/library/autodj/autodjprocessor.cpp

```diff
@@ -163,6 +164,9 @@ AutoDJProcessor::AutoDJProcessor(
 
     m_pCOCrossfader = new ControlProxy("[Master]", "crossfader");
     m_pCOCrossfaderReverse = new ControlProxy("[Mixer Profile]", "xFaderReverse");
+    for (int i = 0; i < 4; i++) {
+        m_pVolumeFader[i] = new ControlProxy(tr("[Channel%1]").arg(i + 1), "volume");
```

Comment:
The key is s programmatically string not for translation. You can use QStringLiteral here.

Reply:
```suggestion
        m_pVolumeFader[i] = new ControlProxy(QStringLiteral("[Channel" )+ QString::number(i + 1) + QStringLiteral("]"), "volume");
```